astronomer/airflow/version_check/update_checks.py

```diff
@@ -81,7 +93,7 @@ def run(self):
             try:
                 update_available, wake_up_in = self.check_for_update()
                 if update_available == UpdateResult.SUCCESS_UPDATE_AVAIL:
-                    self.log.info("A new version of Astronomer Certified Airflow is available")
+                    self.log.info("A new version of %s is available", _app_name(True))
```

Comment:
At this line in the code it is not clear what the `True` does, and I think it should be more explicit.

I suggest in both places it is used, either expand upon it `_app_name(include_airflow=True)` or move the word Airflow into the log `A new version of %s Airflow…`. I am indifferent as to which.

Reply:
Let me use the include_airflow=True because `astronomer runtime airflow` doesn't sound so good to me but `astronomer certified airflow` sounds good